cli/src/command/wallet.rs

```diff
@@ -166,6 +168,16 @@ pub async fn init_command(
         .await?)
 }
 
+pub async fn list_accounts_command(storage_path: &Path, snapshot_path: &Path) -> Result<(), Error> {
+    let password = get_password("Stronghold password", false)?;
+    let wallet = unlock_wallet(storage_path, snapshot_path, password).await?;
+
+    let accounts = wallet.get_account_aliases().await?;
+    println!("{accounts:?}");
```

Comment:
Can we print indexes too?

Reply:
We certainly could (you mean the account index I assume?), but not sure how that would be useful tbh. If there's no alias set, then we assign the account index to it.

Reply:
Sure, but you can access accounts by index so the list of accounts seems like it should include it.

Reply:
hmm, fair point.

Reply:
Do you mind formatting it? Perhaps with a tab between the index and Alias? Column headers would be good too.

Reply:
sure